frontend/packages/dev-console/src/components/project-access/__tests__/ProjectAccess.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import { Formik } from 'formik';
+import { LoadingBox, StatusBox } from '@console/internal/components/utils';
+import ProjectAccess, { ProjectAccessProps } from '../ProjectAccess';
+
+const projectAccessProps: ProjectAccessProps = {
```

Comment:
We are now using `React.ComponentProps` instead of exporting the props type.
Please update the references.

You are also mutating the same instance across multiple tests. You should be creating it new before each test.